monkestation/code/modules/slimecore/slime_color_mutations/mutations/_base_mutation.dm

```diff
@@ -33,6 +33,22 @@
 	if(length(latch_needed))
 		RegisterSignal(host, COMSIG_MOB_FEED, PROC_REF(check_latch))
 
+/// Copies mutation progress to the target slime.
+/// Target should always be a slime of the same type.
+/datum/slime_mutation_data/proc/copy_progress(mob/living/basic/slime/target)
+	if (!istype(host, target))
```

Comment:
i'm unsure what you're trying to accomplish here, but this is a confusing use of istype

Reply:
meh imma just remove it also yep i forgot to do target.type